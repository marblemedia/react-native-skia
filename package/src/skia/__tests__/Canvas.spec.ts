import { ColorType, AlphaType } from "../types";

import { setupSkia } from "./setup";

describe("Canvas", () => {
  it("readPixels", () => {
    const { canvas } = setupSkia();
    const pixels = canvas.readPixels(0, 0, {
      width: 1,
      height: 1,
      colorType: ColorType.RGBA_8888,
      alphaType: AlphaType.Unpremul,
    });
    expect(pixels).toMatchSnapshot();
  });
});
