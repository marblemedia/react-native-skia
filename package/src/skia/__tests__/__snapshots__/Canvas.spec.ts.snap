// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canvas readPixels 1`] = `
Uint8Array [
  0,
  0,
  0,
  0,
]
`;
